### PR TITLE
Feature/tvos

### PR DESCRIPTION
**Pull request by jaelim-ms**
Enabled tvOS targets for Core and Analytics to test on real device.
Keep this change in the code base and work tvOS supports on top of this change.

### CODE REVIEW

File: MobileCenter/MobileCenter/Internals/Device/MSDeviceTracker.m [hunk 1]
```diff
@@ -128,7 +128,7 @@ - (MSDevice *)device {
 - (MSDevice *)updatedDevice {
   @synchronized(self) {
     MSDevice *newDevice = [[MSDevice alloc] init];
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
```
Comment (clpolet) [inline MobileCenter/MobileCenter/Internals/Device/MSDeviceTracker.m]: Do we really need to add this condition? Is possible to have one target that is both iOS and TV OS? It sounds a bit weird :blush:

**Comment by clpolet** [comment]
@jaelim-ms I have changes happening to 2 files while I open the workspace:
```
	modified:   MobileCenter/MobileCenter.xcodeproj/xcshareddata/xcschemes/MobileCenterTV.xcscheme
	modified:   MobileCenterAnalytics/MobileCenterAnalytics.xcodeproj/xcshareddata/xcschemes/MobileCenterAnalyticsTV.xcscheme
```

File: MobileCenter/MobileCenter/Internals/Device/MSDeviceTracker.m (hunk 1 above)
Comment (jaelim-ms) [inline MobileCenter/MobileCenter/Internals/Device/MSDeviceTracker.m]: TARGET_OS_IPHONE is set to 1 in tvOS. This is why.

**Comment by jaelim-ms** [comment]
@clpolet Can you check the scheme issue after pull the branch? It seems be gone.

File: MobileCenter/MobileCenter/Internals/Device/MSDeviceTracker.m (hunk 1 above)
Comment (clpolet) [inline MobileCenter/MobileCenter/Internals/Device/MSDeviceTracker.m]: We should add a TODO here to fix it later if not now.

Comment (jaelim-ms) [inline MobileCenter/MobileCenter/Internals/Device/MSDeviceTracker.m]: If we can take care of it then I can add TODO but it is Apple tvOS SDK side, not our SDK.

**Pull request merged**